app/lib/kaliper/events.rb

```diff
@@ -0,0 +1,48 @@
+require "caliper"
+
+module Kaliper
+  class Events
+
+    attr_accessor :actor
+    attr_accessor :ed_app
+    attr_accessor :group
+    attr_accessor :membership
+    attr_accessor :session
+
+    def initialize(application_instance:, user:)
+      @application_instance = application_instance
+      @user = user
+
+      @actor = Caliper::Entities::Agent::Person.new(
+        id: "#{@application_instance.domain}/users/#{@user.id}",
```

Comment:
Probably need to generate a different user id and include the sis id in the actor